### PR TITLE
Made small improvements to PostgreSQL operations docs.

### PR DESCRIPTION
**Pull request by adamchainz**

### CODE REVIEW

File: docs/ref/contrib/postgres/operations.txt [hunk 1]
```diff
@@ -30,8 +30,8 @@ For example::
             ...
         ]
 
-Django checks that the extension already exists in the database and skips the
-migration if so.
+Django checks if the extension already exists in the database, and if it does,
+it skips adding it again.
```
Comment (knyghty) [inline docs/ref/contrib/postgres/operations.txt:34]: There's a lot of "it" here, referring to the extension, then Django, and finally the extension again. It was a bit confusing to parse. Maybe the second "it" can just be dropped?

File: docs/ref/contrib/postgres/operations.txt [hunk 2]
```diff
@@ -50,7 +50,8 @@ them. In that case, connect to your Django database and run the query
 
 .. class:: CreateExtension(name)
 
-    An ``Operation`` subclass which installs PostgreSQL extensions.
+    An ``Operation`` subclass which installs a PostgreSQL extension. For common
+    extensions, use one of the more specific subclasses below
```
Comment (knyghty) [inline docs/ref/contrib/postgres/operations.txt:54]: Missing a full stop here?

File: docs/ref/contrib/postgres/operations.txt (hunk 1 above)
Comment (adamchainz) [reply docs/ref/contrib/postgres/operations.txt:34]: Yes, one of my worst writing habits. I've rephrased as: "The operation skips adding the extension if it already exists."

File: docs/ref/contrib/postgres/operations.txt (hunk 2 above)
Comment (adamchainz) [reply docs/ref/contrib/postgres/operations.txt:54]: Aye.

**Pull request merged**